ts/src/kraken.ts

```diff
@@ -616,46 +619,40 @@ export default class kraken extends Exchange {
             const quoteId = this.safeString (market, 'quote');
             const base = this.safeCurrencyCode (baseId);
             const quote = this.safeCurrencyCode (quoteId);
-            const darkpool = id.indexOf ('.d') >= 0;
```

Comment:
kraken no longer have any darkpools. it was around 8 years ago, since then they seems to have shut it down